circuits/cpp/src/aztec3/circuits/abis/c_bind.cpp

```diff
@@ -321,6 +321,27 @@ WASM_EXPORT void abis__compute_contract_leaf(uint8_t const* contract_leaf_preima
     NT::fr::serialize_to_buffer(to_write, output);
 }
 
+/**
+ * @brief Generates a siloed commitment tree leaf from the contract and the commitment.
+ *
+ * @param contract_address_buf - The contract address.
+ * @param commitment_buf - The value of the commitment.
+ * @param output - The siloed commitment.
+ * @return WASM_EXPORT
+ */
+WASM_EXPORT void abis__compute_siloed_commitment(uint8_t const* contract_address_buf,
+                                                 uint8_t const* commitment_buf,
+                                                 uint8_t* output)
+{
+    NT::address contract_address;
+    NT::fr commitment;
```

Comment:
I'd prefer if new cbinds are written in msgpack format. This is just
```
CBIND(abis__compute_siloed_commitment, [](NT::address contract_address, NT::fr commitment) {
    return aztec3::circuits::silo_commitment<NT>(contract_address, commitment);
});
```
which actually in this case could be
```
CBIND(abis__compute_siloed_commitment, aztec3::circuits::silo_commitment<NT>);
```

Reply:
After that, remake-bindings will give you TS bindings.